### PR TITLE
minor fixes in tax_smoothing_1.rst

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/tax_smoothing_1.rst [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+
+**Co-author: Sebastian Graves**
+
+
+How to Pay for a War: Part 1
 ============================
 
-An application of Markov jump linear quadratic dynamic programming
+An Application of Markov Jump Linear Quadratic Dynamic Programming
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: can we use `----` for the subheading

File: source/rst/tax_smoothing_1.rst [hunk 2]
```diff
@@ -1,18 +1,30 @@
...
 
-By `Sebastian Graves <https://github.com/sebgraves>`__ and `Thomas J. Sargent <http://www.tomsargent.com/>`__
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
---------------
 
 This notebook constructs generalizations of Barro’s classic 1979 model
 of tax smoothing. Our generalizations are adaptations of extensions of
-his 1979 model suggested by Barro (1999, 2003).
+his 1979 model suggested by Barro (1999, 2003)
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: could you add in citations here

File: source/rst/tax_smoothing_1.rst [hunk 3]
```diff
@@ -22,14 +34,14 @@ Technical tractability induced Barro to assume that
...
--  the one-period risk-free interest rate is constant.
+-  the one-period risk-free interest rate is constant
 
 By using a secret weapon – *Markov jump linear quadratic dynamic
 programming* – we can allow interest rates to move over time in
 empirically interesting ways. Also, by expanding the dimension of the
 state, we can add a maturity composition decision to the government’s
 problem. It is by doing these two things that we extend Barro’s 1979
-model along lines he suggested in Barro (1999, 2003).
+model along lines he suggested in Barro (1999, 2003)
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: cite here

File: source/rst/tax_smoothing_1.rst [hunk 4]
```diff
@@ -22,14 +34,14 @@ Technical tractability induced Barro to assume that
...
 
 Barro (1979) assumed
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: cite here

File: source/rst/tax_smoothing_1.rst [hunk 5]
```diff
@@ -156,19 +166,19 @@ of
...
 matrix Riccati equations that determine :math:`N` optimal value
 functions and :math:`N` linear decision rules. These value functions and
 decision rules apply in the :math:`N` Markov states: i.e., when the
 Markov state is in state :math:`j`, the value function and decision rule
-for state :math:`j` prevails.
+for state :math:`j` prevails
 
-The ordinary discounted linear quadratic dynamic programming problem
+The Ordinary Discounted Linear Quadratic Dynamic Programming Problem
 ''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''''
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: could you change this subheading type to `--`

File: source/rst/tax_smoothing_1.rst [hunk 6]
```diff
@@ -225,19 +235,18 @@ and the matrix :math:`F` in the decision rule for :math:`u_t` satisfies
 
    F = (Q + \beta  B' P B)^{-1} (\beta (B' P A )+ W)
 
-Markov jump coefficients
+Markov Jump Coefficients
 ~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: can this be changed to `^^`

File: source/rst/tax_smoothing_1.rst [hunk 7]
```diff
@@ -319,29 +327,30 @@ and the :math:`F_i` in the optimal decision rules are
    F_i = (Q_i + \beta \sum_j \Pi_{ij} B_i' P_j B_i)^{-1} (\beta \sum_j \Pi_{ij}(B_i' P_j A_i )+ W_i)
 
 Barro (1979) Model
-------------------
+==================
 
 We begin by solving a version of the Barro (1979) model by mapping it
 into the original LQ framework. As mentioned `in this
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: can you delete the `.` and move second sentence to next line -- same for below

File: source/rst/tax_smoothing_1.rst [hunk 8]
```diff
@@ -450,25 +458,25 @@ We can now create an instance of an LQ model:
...
 
- Consequently, taxation is a martingale (:math:`E_t T_{t+1} = T_t`) if:
+Consequently, taxation is a martingale (:math:`E_t T_{t+1} = T_t`) if:
 
 .. math:: (S-MF)(A-BF) = (S-MF)
 
- which holds in this case:
+which holds in this case:
 
 .. code:: ipython3
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: these need to be changed to `.. code-block:: python3`

**Comment by AnjuJoon** [comment]
@natashawatkins  thanks. I will do the suggested changes.

**Comment by AnjuJoon** [comment]
@natashawatkins what should be the Title or the heading of this lecture?

File: source/rst/tax_smoothing_1.rst [hunk 9]
```diff
@@ -19,12 +19,12 @@ How to Pay for a War: Part 1
...
 
 An Application of Markov Jump Linear Quadratic Dynamic Programming
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+------------------------------------------------------------------
 
 
 This notebook constructs generalizations of Barro’s classic 1979 model
 of tax smoothing. Our generalizations are adaptations of extensions of
-his 1979 model suggested by Barro (1999, 2003)
+his 1979 model suggested by Barro (1999, 2003) :cite:`Barro1979`
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: this is the wrong year citation -- you need to look for papers from 1999 and 2003

File: source/rst/tax_smoothing_1.rst [hunk 10]
```diff
@@ -1,18 +1,30 @@
...
-============================
+.. include:: /_static/includes/lecture_howto_py.raw
+
+.. index::
+    single: python
+    
+    
+****************************
+How to Pay for a War: Part 1
+****************************
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:11]: this title is fine

File: source/rst/tax_smoothing_1.rst [hunk 11]
```diff
@@ -1,18 +1,30 @@
...
-An application of Markov jump linear quadratic dynamic programming
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+**Co-author: Sebastian Graves**
 
-By `Sebastian Graves <https://github.com/sebgraves>`__ and `Thomas J. Sargent <http://www.tomsargent.com/>`__
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
---------------
+How to Pay for a War: Part 1
+============================
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: this should be deleted

File: source/rst/tax_smoothing_1.rst [hunk 12]
```diff
@@ -1,18 +1,30 @@
...
+
+An Application of Markov Jump Linear Quadratic Dynamic Programming
+------------------------------------------------------------------
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: this should be changed to `==`

File: source/rst/tax_smoothing_1.rst [hunk 13]
```diff
@@ -53,7 +63,7 @@ Barro (1979) assumed
    :math:`\beta` at which the government discounts future tax disortions
 
 Barro’s model can be mapped into a discounted linear quadratic dynamic
-programming problem.
+programming problem
 
 Our generalizations of Barro’s (1979) model, partly inspired by Barro
 (1999) and Barro (2003), assume
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: citations here

File: source/rst/tax_smoothing_1.rst [hunk 14]
```diff
@@ -1,18 +1,27 @@
...
+    single: python
+    
+    
+****************************
+How to Pay for a War: Part 1
+****************************
+
+.. contents:: :depth: 2
+
+**Co-author: Sebastian Graves**
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: Maybe we could leave the link in to Seb's github as in the original? https://github.com/sebgraves

File: source/rst/tax_smoothing_1.rst [hunk 15]
```diff
@@ -1,18 +1,27 @@
...
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
---------------
+An Application of Markov Jump Linear Quadratic Dynamic Programming
+==================================================================
 
-This notebook constructs generalizations of Barro’s classic 1979 model
+
+This notebook constructs generalizations of Barro’s classic 1979 :cite:`Barro1979` model
 of tax smoothing. Our generalizations are adaptations of extensions of
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: This should be a on new line

File: source/rst/tax_smoothing_1.rst [hunk 16]
```diff
@@ -501,44 +509,44 @@ spending process.
...
 The class also contains a “method”, for simulating the model. This is an
 extension of a similar method in the LQ class, adapted to take into
 account the fact that the model’s matrices depend on the state of the
-world.
+world
 
 Below we import all functionality from this code.
 
 (You should download `the
 file <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:546]: Would you be able to implement this function (https://quanteconpy.readthedocs.io/en/latest/util/notebooks.html) and place the file in QuantEcon.Notebooks

File: source/rst/tax_smoothing_1.rst [hunk 17]
```diff
@@ -501,44 +509,44 @@ spending process.
...
-`here <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__.
+`here <https://github.com/QuantEcon/TaxSmoothing/blob/master/lq_markov.py>`__
 
 The class takes a variable number of arguments, to allow for there to be
 an arbitrary :math:`N` states of the world. To accomodate this, the
 matrices for each state of the world must be held in a “namedtuple”. The
 value and policy functions are then found by iterating on the system of
 algebraic matrix Riccati equations. The solutions for :math:`P,F,\rho`
-are stored in Python “dictionaries”.
+are stored in Python “dictionaries”
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: could you go through these paragraphs and delete the full stops and put sentences on separate lines

File: source/rst/tax_smoothing_1.rst [hunk 18]
```diff
@@ -501,44 +509,44 @@ spending process.
...
 
 The class also contains a “method”, for simulating the model. This is an
 extension of a similar method in the LQ class, adapted to take into
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: ``LQ``

File: source/rst/tax_smoothing_1.rst [hunk 19]
```diff
@@ -501,44 +509,44 @@ spending process.
...
 account the fact that the model’s matrices depend on the state of the
-world.
+world
 
 Below we import all functionality from this code.
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: remove full stop

File: source/rst/tax_smoothing_1.rst [hunk 20]
```diff
@@ -319,29 +324,33 @@ and the :math:`F_i` in the optimal decision rules are
...
-------------------
+==================
+
+We begin by solving a version of the Barro (1979) :cite:`Barro1979` model by mapping it
+into the original LQ framework
 
-We begin by solving a version of the Barro (1979) model by mapping it
-into the original LQ framework. As mentioned `in this
-lecture <http://lectures.quantecon.org/py/perm_income_cons.html>`__, the
+As mentioned `in this lecture <http://lectures.quantecon.org/py/perm_income_cons.html>`__, the
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: we have `:doc:`` ` directive to refer to other lectures

File: source/rst/tax_smoothing_1.rst (hunk 16 above)
Comment (AnjuJoon) [reply source/rst/tax_smoothing_1.rst:546]: I will do

**Comment by AnjuJoon** [comment]
@natashawatkins I will do all the suggested changes.

File: source/rst/tax_smoothing_1.rst [hunk 21]
```diff
@@ -12,7 +12,7 @@ How to Pay for a War: Part 1
 
 .. contents:: :depth: 2
 
-**Co-author: Sebastian Graves <https://github.com/sebgraves> **
+**Co-author**: `Sebastian Graves <https://github.com/sebgraves>`__
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: could you keep it bolded though?

File: source/rst/tax_smoothing_1.rst [hunk 22]
```diff
@@ -23,7 +23,7 @@ This notebook constructs generalizations of Barro’s classic 1979 :cite:`Barro1
 of tax smoothing
 
 Our generalizations are adaptations of extensions of
-his 1979 model suggested by Barro (1999, 2003) 
+his 1979 model suggested by Barro (1999, 2003) :cite:`barro2003religion`
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: can you cite both papers here

File: source/rst/tax_smoothing_1.rst [hunk 23]
```diff
@@ -40,7 +40,7 @@ programming* – we can allow interest rates to move over time in
 empirically interesting ways. Also, by expanding the dimension of the
 state, we can add a maturity composition decision to the government’s
 problem. It is by doing these two things that we extend Barro’s 1979 :cite:`Barro1979`
-model along lines he suggested in Barro (1999, 2003)
+model along lines he suggested in Barro (1999, 2003) :cite:`barro2003religion`
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:44]: cite both papers

File: source/rst/tax_smoothing_1.rst [hunk 24]
```diff
@@ -518,7 +518,7 @@ To implement the extension to the Barro model in which :math:`p_{t,t+1}`
 varies over time, we must allow the M matrix to be time-varying. From
 the mapping of the Barro model into the ``LQ`` framework, this means that
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst]: LQ shouldn't be highlighted here

**Review by natashawatkins — CHANGES_REQUESTED** [review]

File: source/rst/tax_smoothing_1.rst [hunk 25]
```diff
@@ -138,11 +147,11 @@ This notebook describes:
 
 A `sequel to this
 notebook <https://github.com/QuantEcon/TaxSmoothing/blob/master/Tax_Smoothing_2.ipynb>`__
-describes applies Markov LQ control to settings in which a government
+describes applies Markov ``LQ`` control to settings in which a government
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:150]: this shouldn't be highlighted

File: source/rst/tax_smoothing_1.rst [hunk 26]
```diff
@@ -138,11 +147,11 @@ This notebook describes:
 
 A `sequel to this
 notebook <https://github.com/QuantEcon/TaxSmoothing/blob/master/Tax_Smoothing_2.ipynb>`__
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:149]: can you use `:doc:`` ` to refer to other lectures

File: source/rst/tax_smoothing_1.rst [hunk 27]
```diff
@@ -355,16 +366,16 @@ while :math:`b_{t-1,t}` is an endogenous state variable inherited from
 the past at time :math:`t` and :math:`p_{t,t+1}` is an exogenous state
 variable at time :math:`t`. To begin with, we will assume that
 :math:`p_{t,t+1}` is constant (and equal to :math:`\beta`), but we will
-also extend the model to allow this variable to evolve over time.
+also extend the model to allow this variable to evolve over time
 
-To map into the LQ framework, we will use
+To map into the ``LQ`` framework, we will use
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:371]: shouldn't be highlighted

File: source/rst/tax_smoothing_1.rst [hunk 28]
```diff
@@ -437,47 +447,47 @@ that :math:`G_t` follows an AR(1) process:
     Q = M.T @ M
     W = M.T @ S
     
-    # Small penalty on debt required to implement no-ponzi scheme
+    # Small penalty on debt required to implement no-Ponzi scheme
     R[0, 0] = R[0, 0] + 1e-9
 
-We can now create an instance of an LQ model:
+We can now create an instance of an ``LQ`` model:
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:453]: shouldn't be highlighted

File: source/rst/tax_smoothing_1.rst [hunk 29]
```diff
@@ -437,47 +447,47 @@ that :math:`G_t` follows an AR(1) process:
...
+.. code-block:: python3
 
     LQBarro = qe.LQ(Q, R, A, B, C=C, N=W, beta=β)
     P, F, d = LQBarro.stationary_values() 
     x0 = np.array([[100, 1, 25]])
 
 We can see the isomorphism by noting that consumption is a martingale in
 the permanent income model, and that taxation is a martingale in Barro’s
-model. We can check this using the F matrix of the LQ model. As
+model. We can check this using the :math:`F` matrix of the ``LQ`` model. As
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:463]: new line + shouldn't be highlighted

File: source/rst/tax_smoothing_1.rst [hunk 30]
```diff
@@ -489,9 +499,9 @@ Barro model a large number of times:
 
 We can see a similar, but smoother pattern, if we plot government debt
 over time. Debt is smoother due to the persistence of the government
```
Comment (natashawatkins) [inline source/rst/tax_smoothing_1.rst:501]: new line

**Pull request merged**